packages/framer-motion/src/value/index.ts

```diff
@@ -34,6 +35,12 @@ export class MotionValue<V = any> {
      */
     version = "__VERSION__"
 
+    /**
+     * If a MotionValue has an owner, it was created internally within Framer Motion
+     * and therefore has no external listeners. It is therefore safe to animate via WAAPI.
+     */
+    owner?: VisualElement
```

Comment:
Shouldn't we block or error adding external listeners?

Reply:
No, it's not a problem if there are external listeners, it just means we run animations as before